charts/postgres-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: postgres-operator
-version: 1.13.0
```

Comment:
this will be bumped to 1.14.0. We have no branching in place to create a bugfix release for 1.13.0. Too many feature already merged to main trunk.

Reply:
I wanted to keep it at 1.13.0 for now, but ok, we can also set to 1.14.0.